test/game/game.js

```diff
@@ -263,4 +266,58 @@ describe('Game Server', () => {
       });
     });
   });
+
+
+  it('Should change send message to other player when a message is emitted', (done) => {
```

Comment:
Good job including tests. Do you think this test is sufficient to test that our feature meats requirements?